### ISSUE TITLE
Error while unarchiving an experiment

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#235]
In GitLab by @ jlope2 on Jan 27, 2017, 12:53

Reported by @vsicardi 

```
[CRITICAL] The directory /esnas/autosubmit/m04h is needed and does not exist.
[WARNING] Does an experiment with the given id exist?
```

The experiment' folder checking was wrongly added to this action since version 3.6.0.

It shouldn't be checked.

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Jan 27, 2017, 12:57

Mentioned in commit 1de8779dea3e81ffa8b7f9e035bcabf4e9d7b225

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Jan 27, 2017, 13:04

Hi @vsicardi,

The bug is now fixed (1de8779d) and it will be available since next release.

If you need to *UNARCHIVE* an experiment, please load the version ```3.5.0```, unarchive it, and reload (*module load*) the current version of Autosubmit to continue working with your unarchived experiment.

This bug was stupidly introduced by myself, sorry. Fortunately it only affects the action of unarchiving an experiment.

Thanks!

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 15:18

Mentioned in commit 7c958a58673d3e13ebb65e62fe47285c7c034744

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 15:29

Mentioned in commit d53129298d8fbc3519aa0714523a398c1a0ec0dc

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 18:17

Fixed! 

Available since Autosubmit v3.7.8.
